crmsh/parse.py

```diff
@@ -653,6 +655,75 @@ def match_arguments(self, out, name_map, implicit_initial=None, terminator=None)
             else:
                 break
 
+        self.complete_advised_ops(out)
+
+    def complete_advised_ops(self, out):
+        """
+        Complete operation actions advised values
+        """
+        if not self.complete_op_advised or out.tag != "primitive":
+            return
+        ra_inst = ra.RAInfo(out.get('class'), out.get('type'), out.get('provider'))
+        ra_actions_dict = ra_inst.actions()
+        if not ra_actions_dict:
+            return
+
+        def extract_advised_monitor_value(advised_dict, attr, role=None):
+            adv_attr_value = None
+            try:
+                if role:
+                    for monitor_item in advised_dict['monitor']:
+                        if role == monitor_item['role']:
+                            adv_attr_value = monitor_item[attr]
+                else:
+                    adv_attr_value = advised_dict['monitor'][0][attr]
+            except KeyError:
+                pass
+            return adv_attr_value
+
+        action_advised_attr_dict = {k:v for k, v in ra_actions_dict.items() if k in constants.ADVISED_ACTION_LIST}
+        operations_node = out.find("operations")
+        configured_action_list = []
+        # no operations configured
+        if operations_node is None:
+            operations_node = xmlutil.child(out, 'operations')
+        # has operations configured
+        else:
+            op_nodes_list = operations_node.findall("op")
+            for op_node in op_nodes_list:
+                action = op_node.get('name')
+                # complete advised value if interval or timeout not configured
+                if action == "monitor":
+                    adv_interval = extract_advised_monitor_value(action_advised_attr_dict, 'interval', op_node.get('role')) or \
+                            constants.DEFAULT_INTERVAL_IN_ACTION
+                    adv_timeout = extract_advised_monitor_value(action_advised_attr_dict, 'timeout', op_node.get('role')) or \
+                            constants.DEFAULT_TIMEOUT_IN_ACTION
```

Comment:
OTOH, `timeout` is an optional attribute, meaning pacemaker has an internal default (20s), or users could have their own global default defined in `op_defaults` section. So we don't really need yet another crmsh defined default here.

Reply:
I think it's no harm to keep this?:)

Reply:
In case `timeout` is not set by the user meanwhile no value is advised in the metadata, we'd like omit the setting rather than taking a crmsh-default `DEFAULT_TIMEOUT_IN_ACTION`.